CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add support to configure views when creating MeterProvider using the config package. (#5654)
 - Add log support for the autoexport package. (#5733)
 - Add support for disabling the old runtime metrics using the `OTEL_GO_X_DEPRECATED_RUNTIME_METRICS=false` environment variable. (#5747)
+- Add support for signal-specific protocols environment variables for the autoexport package. (#5807)
```

Comment:
```suggestion
- Add support for signal-specific protocols environment variables in `go.opentelemetry.io/contrib/exporters/autoexport`. (#5816)
```

Maybe we should also specify the exact variables we support.

Reply:
Yes, it'll be better. I've added them.